main.tf

```diff
@@ -1,8 +1,8 @@
 module "lambda_role" {
-  source           = "github.com/traveloka/terraform-aws-iam-role.git//modules/service?ref=v0.2.0"
-  aws_service      = "lambda.amazonaws.com"
-  role_identifier  = "${var.lambda_name}"
-  role_description = "Role for lambda ${var.lambda_name}"
+  source = "github.com/traveloka/terraform-aws-iam-role.git//modules/lambda?ref=v0.4.0"
+
+  product_domain   = "${var.product_domain}"
```

Comment:
This needs to be product_domain|service_name.

Reply:
I believe the module supports either one, you just need to pass either product domain value, or service name. So if this Lambda is part of the service, then pass service name, otherwise product domain only can be used.

In general being more specific in naming is better than not, so service name is preferable.